logic_fuctions.py

```diff
@@ -0,0 +1,28 @@
+def calculate_winner(squares):
+    lines = [
+        [0, 1, 2],
+        [3, 4, 5],
+        [6, 7, 8],
+        [0, 3, 6],
+        [1, 4, 7],
+        [2, 5, 8],
+        [0, 4, 8],
+        [2, 4, 6],
+    ]
+
+    for line in lines:
+        if line[0] == line[1] and line[0] == line[2]:
+            return line[0]
+    return None
+
+
+def get_fresh_board():
+    fill = [None for _ in range(10)]
+
+
+def get_next_board(squares, i, x_is_next):
+    if squares[i] == "X":
+        x_is_next = "O"
+    else:
+        x_is_next = "X"
+    return x_is_next
```

Comment:
Can you push and link to the code this is supposed to port?  This function looks wrong.

Reply:
is it because x_is_next needs to be xIsNext?

Reply:
Not that.  Could you get the changes we have made in the react app pushed to github.

Reply:
i just pushed them

Reply:
The snake casing here is correct, but this function isn't doing what this https://github.com/aepaysinger/react-tic-tac-toe/pull/1/files#diff-8f16384a8b1758c444d336f8516112ea3b4322fd4ae66893c2edbeeae1ae18b5R23 function does.